src/components/CopilotUsagePanel.css

```
@@ -711,6 +711,15 @@
   position: relative;
 }
 
+.usage-budget-progress-native {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)